docs/Releases/v0.2.0-release.md

```diff
@@ -0,0 +1,364 @@
+$ kubectl get packages 
+NAME                                                     AGE
+operatorhubio-ack-acm-controller                         3m12s
+operatorhubio-ack-apigatewayv2-controller                3m12s
+operatorhubio-ack-applicationautoscaling-controller      3m12s
+operatorhubio-ack-cloudtrail-controller                  3m12s
+operatorhubio-ack-dynamodb-controller                    3m12s
+operatorhubio-ack-ec2-controller                         3m12s
+operatorhubio-ack-ecr-controller                         3m12s
+operatorhubio-ack-eks-controller                         3m12s
+operatorhubio-ack-elasticache-controller                 3m12s
+operatorhubio-ack-emrcontainers-controller               3m12s
+operatorhubio-ack-eventbridge-controller                 3m12s
+operatorhubio-ack-iam-controller                         3m12s
+operatorhubio-ack-kinesis-controller                     3m12s
+operatorhubio-ack-kms-controller                         3m12s
+operatorhubio-ack-lambda-controller                      3m12s
+operatorhubio-ack-memorydb-controller                    3m12s
+operatorhubio-ack-mq-controller                          3m12s
+operatorhubio-ack-opensearchservice-controller           3m12s
+.
+.
+.
+
+$ kubectl get bundlemetadata 
+NAME                                                            AGE
+operatorhubio-ack-acm-controller.v0.0.1                         3m58s
+operatorhubio-ack-acm-controller.v0.0.2                         3m58s
+operatorhubio-ack-acm-controller.v0.0.4                         3m58s
+operatorhubio-ack-acm-controller.v0.0.5                         3m58s
+operatorhubio-ack-acm-controller.v0.0.6                         3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.10               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.11               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.12               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.13               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.14               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.15               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.16               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.17               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.18               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.19               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.20               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.21               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.22               3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.0.9                3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.1.0                3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.1.1                3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.1.2                3m58s
+operatorhubio-ack-apigatewayv2-controller.v0.1.3                3m58s
+.
+.
+.
+```
```

Comment:
Nit: Break these out into sections that explain a bit better what a `Package` and `BundleMetdata` are. What if I have no prior knowledge of the OLMv0 `PackageManifest` API? How do I know what these *new* APIs are supposed to represent?

Reply:
My argument is, if you're not familiar with OLMv0, do you really care about these milestone release notes? Or would you rather just wait for the GA OLM v1 user docs to start using OLM v1?

Reply:
While I don't think that is an unreasonable argument I feel like that assumption gate keeps early adopters of this project to being only those that have used OLMv0 and understand the concepts. IMO if we are providing any form of guide in the docs (including release notes) it should be done in a way that anyone can get an understanding of the concepts being discussed (even if it is a link to another doc to explain them in more detail). If done in this way, it allows for those that have "stumbled" across the repository to explore and get a base understanding of what the project does and why (I put stumbled in quotes because it's likely that if they found the project they were looking around for solutions to solve a particular use case that lead them to finding the project). These potential users may not be interested in how it is better than OLMv0 and just care about using the new stuff even if it isn't GA.

That being said, this isn't a hill I'm willing to die on - just wanted to share my thought process on this.